starcli/search.py

```diff
@@ -183,8 +201,6 @@ def search(
         print("DEBUG: auth: off")
 
     request = get_valid_request(url, auth)
-    if request is None:
-        return request
 
     return request.json()["items"]
```

Comment:
Suggest adding `return None` if `request` is None, because when user provides an invalid credential or some other error occurs, a message will be printed in `get_valid_request`, but since None has no attribute json, a error when be thrown with trace back
```suggestion
    return request.json()["items"] if request else None
```
Also, it worked fine without removing lines 186 to 187 before this. Wonder why you removed it